pkg/kev/converter/kubernetes/transform.go

```diff
@@ -1209,6 +1212,36 @@ func (k *Kubernetes) configEnvs(projectService ProjectService) ([]v1.EnvVar, err
 					"path":            path,
 				}, "Unsupported Pod field reference: %s", path)
 			}
+		case "container":
+			// Selects a resource of the container. Only resources limits and requests are currently supported:
+			// 		limits.cpu, limits.memory, limits.ephemeral-storage,
+			//  	requests.cpu, requests.memory and requests.ephemeral-storage
+			resources := []string{
+				"limits.cpu", "limits.memory", "limits.ephemeral-storage",
+				"requests.cpu", "requests.memory", "requests.ephemeral-storage",
+			}
+			resource := strings.Join(parts[2:], ".")
+
+			sort.Strings(resources)
+			i := sort.SearchStrings(resources, resource)
+			if i < len(resources) && resources[i] == resource {
```

Comment:
Similar comment to the Pod env var implementation in #258 ...
See: https://github.com/appvia/kev/pull/258#discussion_r487675087